src/app/static/src/app/store/surveyAndProgram/actions.ts

```diff
@@ -151,6 +164,7 @@ export const actions: ActionTree<SurveyAndProgramState, RootState> & SurveyAndPr
             .delete("/disease/anc/")
             .then(() => {
                 commit({type: SurveyAndProgramMutation.ANCUpdated, payload: null});
+                commitClearGenericChartDataset(commit, DATASET_TYPE.ART)
```

Comment:
This doesn't look right... you'll need a test that the payloads are correct here. Please see my [previous comment](https://github.com/mrc-ide/hint/pull/547#discussion_r713972333).